examples/confluent_cloud.py

```diff
@@ -87,7 +87,8 @@ def acked(err, msg):
     'sasl.username': '<ccloud key>',
     'sasl.password': '<ccloud secret>',
     'group.id': str(uuid.uuid1()),  # this will create a new consumer group on each invocation.
-    'auto.offset.reset': 'earliest'
+    'auto.offset.reset': 'earliest',
+    'enable.partition.eof': True
```

Comment:
do we really want eof-checking in this example, which is likely to be widely used?

Reply:
I left any example that does something with the error. We can remove it though.

Reply:
Let's create a new example (later) that does EOF reading, to keep these examples clean.